docs/release-notes.md

```diff
@@ -1,5 +1,38 @@
 # Release notes
 
+## Trino Gateway 10 (?? July 2024)
+
+[JAR file gateway-ha-10-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/10/gateway-ha-10-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:10`
+
+* [⚠](#breaking) Remove support for any Dropwizard and Jetty Proxy integration 
+  and usage. Add [Airlift](https://github.com/airlift/airlift) as base 
+  application framework as used in Trino. This changes the supported Trino 
+  Gateway configuration and relevant properties. Find details in the
+  documentation, and specifically refer to the
+  [upgrade guide](migration-to-airlift.md) when migrating from older releases. 
+  ([#41](https://github.com/trinodb/trino-gateway/issue/41)),
+  ([#317](https://github.com/trinodb/trino-gateway/pull/317)),
+  ([#382](https://github.com/trinodb/trino-gateway/pull/382)),
```

Comment:
I think simply linking to the issue is sufficient.

Reply:
I removed the two PR links and added a comment in the issue instead.